internal/core/src/query/visitors/ExecPlanNodeVisitor.cpp

```diff
@@ -123,7 +124,20 @@ ExecPlanNodeVisitor::VectorVisitorImpl(VectorPlanNode& node) {
                            timestamp_,
                            final_view,
                            search_result);
-
+    if (search_result.iterators.has_value()) {
+        GroupBy(search_result.iterators.value(),
+                node.search_info_,
+                search_result.group_by_values_,
+                *segment,
+                search_result.seg_offsets_,
+                search_result.distances_);
+        AssertInfo(search_result.seg_offsets_.size() ==
```

Comment:
It's enabled to format string with `AssertInfo/PanicInfo` like:
```
AssertInfo(a == b, "{} != {}", a, b)
```

Reply:
changed